server.go

```diff
@@ -674,10 +674,17 @@ func (sp *serverPeer) handleServeGetData(invVects []*wire.InvVect,
 			continue
 		}
 		if dataMsg == nil {
+			// Keep track of all items that were not found in order to send a
+			// consolidated messsage once the entire batch is processed.
 			if notFoundMsg == nil {
 				notFoundMsg = wire.NewMsgNotFound()
 			}
 			notFoundMsg.AddInvVect(iv)
```

Comment:
unrelated, but we might want to comment on why this error is ignored (obviously we can't exceed the total amount of invs this message can hold, if we are responding to a proper getdata message in the first place)